src/databrowser.js

```diff
@@ -165,7 +165,7 @@ define([
                 }
                 info = info._parent;
             }
-            vellum.handleDropFinish(target, path);
+            vellum.handleDropFinish(target, path, widget.mug);
```

Comment:
https://github.com/dimagi/Vellum/blob/master/src/core.js#L725-L745

Agreed it's not the best. It has always passed in mug and i just used that to use form.